src/nls/fr/shortcuts-mac.json

```diff
@@ -32,7 +32,8 @@
         },
         "LAYER": {
             "CREATE_ARTBOARD": "b",
-            "DUPLICATE": "j"
+            "DUPLICATE": "j",
+            "NEW_VECTOR_MASK_FROM_SHAPE": "m"
```

Comment:
I think it is best to _not_ add shortcuts to the non-English dictionaries. If a key is missing, it will fall back to English automatically, and editing these files directly might be confusing for the loc team.